xtimeutil/remapper.py

```diff
@@ -199,13 +208,45 @@ def get_outgoing_time_axis_info(self):
         x.update(attrs)
         return x
 
+    def mean(self, data):
+        if isinstance(data, xr.DataArray):
+            indata = _sanitize_input_data(
+                data, self.incoming['time_coord_name'], self.weights, self.incoming
+            )
+            if isinstance(indata.data, dask.array.Array):
+                incoming_time_chunks = dict(zip(indata.dims, indata.chunks))[
+                    self.incoming['time_coord_name']
+                ][0]
+                outdata = _mean(
+                    self.weights.chunk({_OUTGOING_KEY: incoming_time_chunks}),
+                    indata,
+                    self.incoming['time_coord_name'],
+                )
+            else:
+                outdata = _mean(self.weights, indata, self.incoming['time_coord_name'])
+            outdata = _finalize_output_data(
+                outdata, self.incoming['time_coord_name'], self.outgoing, self.incoming
+            )
+            return outdata
+
+        else:
+            raise NotImplementedError
```

Comment:
Might be good to tell people *why* they are getting a `NotImplementedError` (namely, what isn't implemented?):

```suggestion
            raise NotImplementedError('Currently only works on xarray DataArrays')
```